parts/k8s/cloud-init/artifacts/docker-monitor.service

```diff
@@ -1,6 +1,10 @@
 [Unit]
 Description=a script that checks docker health and restarts if needed
+{{- if NeedsContainerd}}
+After=containerd.service
```

Comment:
Wouldn't this be the same as writing:
```
After={{GetContainerRuntime}}.service
```
Or is there some additional disconnect between the two?  Right now I see those as the same.

Reply:
It is. The likelihood of the containerRuntime allowed values changing (from "docker" and "containerd" to something else to indicate the same configuration) is 0%. The likelihood that the apt package would rename the current service names to something else is perhaps a mite larger, but also unlikely, so your more elegant suggestion will work.

In the event the coupling between the CRI name and the service name goes away in the future, we can refactor to the more verbose if/else expression.

Reply:
Worst case we could have GetContainerRuntimeService in addition to GetContainerRuntime

/lgtm